DOCSP-29461: node lab embedding (#820)
mongodb/docs-node: source/fundamentals/crud/write-operations/modify.txt

Overview
--------

You can modify documents in a MongoDB collection using **update**

> and **replace** operations. Update operations change

Reviewer: Issue:
I think this sentence might be inaccurate as it mentions updating fields but omits values, which is probably the more common use case.

Suggestion:
"Update operations modify the fields and values of a document while keeping the other fields and values."

specified fields in one or more documents and leave other fields and values

> unchanged. Replace operations remove all fields except the ``_id`` field
> in an existing document and substitute them with specified fields and values.

Reviewer: Suggestion:
I think if the prior suggestion is taken, this sentence could provide a parallel structure to make it easier to compare the two.

```suggestion
unchanged. Replace operations substitute the document fields and values with the specified replacement document while keeping the ``_id`` value.
```
